File: Domain/LabResult.cs
using System;

namespace Domain
{
    public class LabResult
    {
        public Guid Id { get; set; }

        public string Sample { get; set; }

        public string ProblemProportion { get; set; }

        public DateTime Date {get; set;}

        public string Result { get; set; }

        public string ResultProportion { get; set; }

        public string status {get; set;}

        public Patient patient { get;set;}

        public Doctor doctor {get; set;}

    }
}

File: client-app/src/app/models/labresult.ts
import { Doctor } from "./doctor";
import { Patient } from "./patient";

export interface LabResult {
    id: string;
    sample: string;
    problemProportion: string;
    date: string;
    result:string;
    resultProportion: string;
    status: string;
    patient:Patient;
    doctor: Doctor;
}
